README.md

```diff
@@ -22,9 +22,12 @@ In macOS 12.3, Apple removed the Python 2.7 install. Out-of-the-box, there is no
 
 Some options for providing an appropriate Python:
 
-1) If you also use Munki, use Munki's bundled Python. You could make a symlink at /usr/local/bin/python pointing to /usr/local/munki/munki-python (this assumes /usr/local/bin is in your PATH, which it is by default. You could create symlink in any writable directory in your PATH if it differs)
+1) If you also use Munki, use Munki's bundled Python. You could make a symlink at /usr/local/bin/python pointing to /usr/local/munki/munki-python (this assumes /usr/local/bin is in your PATH, which it is by default. You could create the symlink in any writable directory in your PATH if it differs)
 2) Install Python from https://www.python.org. You might still need to create a symlink somewhere so that `/usr/bin/env python` executes the Python you installed.
-3) Install Apple's Python 3 by running `/usr/bin/python3` and accepting the prompt to install Python (if Xcode or the Command line development tools are not already present). Again you might need to create a symlink so that `/usr/bin/env python` executes the Python you installed.
+3) Install Apple's Python 3 by running `/usr/bin/python3` and accepting the prompt to install Python (if Xcode or the Command line development tools are not already present). Again you might need to create a symlink so that `/usr/bin/env python` executes the Python you installed:
+```
+sudo ln -s /Library/Developer/CommandLineTools/usr/bin/python3 /usr/local/bin/python
+```
```

Comment:
Is this the "correct" symlink if you've installed Xcode but _not_ the command line tools? I don't think it is. The generic approach would be `sudo ln -s /usr/bin/python3 /usr/local/bin/python`.

Reply:
Doing that exact path WITHOUT full Xcode installed on Ventura just earlier caused reinstall prompts like you would see with xcode-select, which loop and don't allow the current env shebang to work in my testing. ![image](https://user-images.githubusercontent.com/388808/200093579-fb03646b-1d1b-497c-9bfc-c6821cfaae6d.jpeg)
```
% sudo ln -s /usr/bin/python3 /usr/local/bin/python
% env python                                       
xcode-select: Failed to locate 'python', requesting installation of command line developer tools
```
The exact verbiage leading up to this line implies a CLI tools-only install. I think this later python option being explicit conveys a hands-off 'each case requires handling' but the ways just standalone Xcode CLI tools get on computers these days feels least effort for lazy sysadmins. Folks may install homebrew (which can trigger the xcode-select delivery method) or have developer account downloads access which still leaves us with this exact path being correct as far as I can tell.